### PR TITLE
Upgrades should now update upon clicking reset button

### PR DESCRIPTION
**Pull request by Dan-Quin**
Just had the reset button click dispatch a function that would run to check the starter upgrades, after testing from me and Leo this seems to work consistently.

### CODE REVIEW

File: frontend/src/game/PhaserGame.tsx [hunk 1]
```diff
@@ -6,7 +6,9 @@ import { useDispatch, useSelector } from "react-redux";
 import { addBits } from "../slices/gameDataSlice";
 import { IGameData, GameVariable } from "../../../shared/types";
 import { calculateVariableValue } from "../../../shared/util";
+import upgradesSlice from "../slices/upgradesSlice";
```
Comment (josephmaxwellhyland) [inline frontend/src/game/PhaserGame.tsx:9]: This import is not used

File: frontend/src/slices/upgradesSlice.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { createSlice } from "@reduxjs/toolkit";
 import { IUpgrade } from "../../../shared/types";
 import { starterUpgrades } from "../../../shared/upgrades";
+import { resetGameData } from "./gameDataSlice";
```
Comment (josephmaxwellhyland) [inline frontend/src/slices/upgradesSlice.ts:4]: These two imports are not used

**Review by josephmaxwellhyland — CHANGES_REQUESTED** [review]
Just remove a few unused imports and this should be good to go!

File: frontend/src/slices/upgradesSlice.ts (hunk 2 above)
Comment (Dan-Quin) [reply frontend/src/slices/upgradesSlice.ts:4]: Both imports have been deleted and taken care of.

File: frontend/src/game/PhaserGame.tsx (hunk 1 above)
Comment (Dan-Quin) [reply frontend/src/game/PhaserGame.tsx:9]: Both imports have been removed.

**Review by josephmaxwellhyland — APPROVED** [review]
Looks good to me!

**Pull request merged**